app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt

```diff
@@ -130,6 +136,16 @@ class CompletedStoryListActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testFAQListActivity_hasCorrectActivityLabel() {
```

Comment:
The name of this test case is incorrect.

Reply:
On it